Doc/library/sys.rst

```diff
@@ -1060,6 +1062,31 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+.. data:: module_names
+
+   A tuple of strings giving the names of standard library modules.
+
+   The list is the same on all platforms. Modules which are not available on
```

Comment:
```suggestion
   The output is the same on all platforms. Modules which are not available on
```

"the list" can be confused with the return type (although we say before is a tuple).